pallets/token-wrapper/src/lib.rs

```diff
@@ -143,10 +143,15 @@ pub mod pallet {
 	pub type WrappingFeePercent<T: Config> =
 		StorageMap<_, Blake2_128Concat, T::AssetId, BalanceOf<T>, OptionQuery>;
 
+	/// Fee recipient, account which will be receiving wrapping cost fee.
+	#[pallet::storage]
+	#[pallet::getter(fn fee_recipient)]
+	pub(super) type FeeRecipient<T: Config> = StorageValue<_, T::AccountId>;
+
 	/// The proposal nonce used to prevent replay attacks on execute_proposal
 	#[pallet::storage]
 	#[pallet::getter(fn proposal_nonce)]
-	pub type ProposalNonce<T: Config> = StorageMap<_, Blake2_128Concat, Vec<u8>, T::ProposalNonce>;
+	pub type ProposalNonce<T: Config> = StorageValue<_, T::ProposalNonce, ValueQuery>;
```

Comment:
This should not be a value, it should be a map no? Since the token wrapper pallet maintains many tokens at once, it should be a map for proposal nonces for each token.

Reply:
We could make it a map from ResourceId to nonce, since that would capture the differences in a proposal targetting a specific token resource.

Reply:
I see we only have 1 main feeRecipient too, which may want to be updated as well.